src/modules/AnalysisExecution/components/SubmissionCodeViewer/presenter.jsx

```diff
@@ -21,35 +21,12 @@
  */
 
 import React from 'react';
-import BEMHelper from 'services/BemHelper';
-import { PageContent, LoadingPanel } from 'arachne-ui-components';
-import MediaViewer from 'components/MediaViewer';
-import Toolbar from './Toolbar/index';
-
-require('./style.scss');
-
-function SubmissionCodeViewer({ language, isLoading, pageTitle, downloadLink, mimeType, content, name, title, createdAt }) {
-  const classes = new BEMHelper('submission-code');
+import FileViewer from 'components/FileViewer';
 
+function SubmissionCodeViewer(props) {
   return (
-    <PageContent title={pageTitle}>
-      <div {...classes()}>
-        <Toolbar />
-        <div {...classes('content')}>
-          <MediaViewer
-            language={language}
-            mimeType={mimeType}
-            data={content}
-            downloadLink={downloadLink}
-            name={name}
-            title={title}
-            createdAt={createdAt}
-          />
-        </div>
-        <LoadingPanel active={isLoading} />
-      </div>
-    </PageContent>
+    <FileViewer params={props}/>
```

Comment:
I would clarify, what params are passed down (so specify them explicitly, not as batch). Because later, when you want to change smth in container (mapState / mapDispatch), you won't be available to quickly find out, whether the thing is used and how is it used